app/src/app/api/importer/[slug]/source-data/route.ts

```diff
@@ -0,0 +1,35 @@
+import minioClient from "@/lib/minioClient";
+import { getTemporalWorkflowClient } from "@/lib/temporalClient";
+import { NextRequest, NextResponse } from "next/server";
+import { ImporterStatus } from "../ImporterDto";
+
+export async function GET(
+  _req: NextRequest,
+  { params }: { params: { slug: string } },
+  res: NextResponse
+) {
+  const { slug: importerId } = params;
+  const client = getTemporalWorkflowClient();
+  const handle = client.getHandle(importerId);
+  const result = await handle.query<ImporterStatus>("importer:status");
```

Comment:
ich vermute, das kommt nicht in der status query, sondern es gibt eine dedizierte sourceData query.

Reply:
Eigentlich sollte das schon in den Status, ich mach mal ein Issue dafür auf